packages/cyberstorm/src/components/PackageSearch/PackageSearch.tsx

```diff
@@ -53,20 +55,29 @@ export function PackageSearch(props: Props) {
 
       <div className={styles.contentWrapper}>
         <div className={styles.sidebar}>
-          <SectionMenu
-            allSections={allSections}
-            selected={section}
-            setSelected={setSection}
-          />
+          <Suspense
```

Comment:
This still suffers from the same problem as the previous version of this PR. `allSections` and `categories` are loaded outside the suspense, so the fallback is never triggered.

BUT it's also a bit different case, since it *seems to me* the values are *currently* not loaded on the client side at all, only the SSR content is used. Which may or may not be what we actually want (better performance with cache vs. new categories taking time to be shown on the site). *If* we are happy with the SSR-only, this `Suspense` is probably not needed at all.

Reply:
Ah, truuue. I just checked the designs and added skeletons to where ever they were in designs.

IMO the sidebar doesn't need a skeleton.